build/cli.js

```diff
@@ -282,6 +286,9 @@ program
   .option("-i, --interactive", "Ask what to do when encountering flaws", {
     default: false,
   })
+  .option("-n, --nohtml", "Do not build index.html", {
```

Comment:
The reason `--no-html` didn't work is because caporal can take any flag and negate it. E.g. if `--peter` is a valid flag, so is `--no-peter`. [click](https://click.palletsprojects.com/en/8.0.x/) does the same thing. I don't know why it's not mentioned in the Caporal documentation. 

What do you think about https://github.com/mdn/yari/pull/4258/files as an alternative implementation. 
Then you get the choice between `--html` and `--no-html` and the default is the same as typing `--html` which is what we want. Is that any cleaner?

Reply:
I think using `--html` and `--no-html` makes it more confusing. Thanks for the explanation why it didn't work. I'd prefer this version.

Reply:
I linked to the wrong PR. It was supposed to be https://github.com/fiji-flo/yari/pull/312
I closed it and linked to your comment. Thanks for considering it.